packages/expo/src/ScreenOrientation/ScreenOrientation.ts

```diff
@@ -84,7 +56,7 @@ export async function lockPlatformAsync(options: PlatformOrientationInfo): Promi
   }
 
   const { screenOrientationConstantAndroid, screenOrientationArrayIOS } = options;
-  let platformOrientationParam;
+  let platformOrientationParam: number | Orientation[] | undefined;
```

Comment:
This might not hold true for future platforms where we want to use the same API layer... -> any